### ISSUE TITLE
rcedit with wine crashes on non-ascii characters

### ISSUE THREAD

**Issue by Eusebius1920** [electron/node-rcedit#9]
If you use an executable (.exe) with an Umlaut (ä,ö,ü) or with an accent (é, è, etc) in its name, then node-rcedit crashes (at least if used with wine on OSX)

The Error is as follows:
```
fixme:heap:HeapSetInformation 0x0 1 0x0 0
Fatal error: Unable to load file
```

wine: wine-1.6.2
rcedit: 0.3.0

You should at least mention this in the documentation.

**Comment by shenlq** [comment]
thanks @Eusebius1920

**Comment by yesmeck** [comment]
It's creashed even on windows if `exe` file has non-ascii name.

**Comment by phaibin** [comment]
+1 on OS X El Capitan with wine-1.9.7 (Staging) & rcedit: 0.3.0. I think the root cause is from Wine. So sad.

**Comment by kevinsawicki** [comment]
I think this has been addressed by https://github.com/electron/rcedit/pull/23 and is fixed in the latest release, 0.6.0
